<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSDlgHealthManagerSetting</class>
 <widget class="QDialog" name="CSDlgHealthManagerSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Health Manager Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_IP">
       <property name="text">
        <string>Server IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_port">
       <property name="text">
        <string>Server port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_port"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_IP"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_period">
       <property name="text">
        <string>Send Period</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_period"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_unit">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
